src/clinical/clinical-service.ts

```diff
@@ -200,6 +204,21 @@ export const getPaginatedClinicalData = async (programId: string, query: Clinica
   return data;
 };
 
+export const getClinicalSearchResults = async (programId: string, query: ClinicalSearchQuery) => {
+  if (!programId) throw new Error('Missing programId!');
+  const start = new Date().getTime() / 1000;
+
+  // Get list of donorIds + submitterDonorIds matching search results
+  const { donors } = await donorDao.findByProgramDonorSearch(programId, query);
+
+  const data = filterDonorIdDataFromSearch(donors as Donor[], query);
+
+  const end = new Date().getTime() / 1000;
+  L.debug(`getPaginatedClinicalData took ${end - start}s`);
```

Comment:
Just in case you didn't know this exsited:
``` js
console.time('findDonorsByProgram'); // starts a timer named 'findDonorsByProgram'

// do work

console.timeLog('findDonorsByProgram'); // console log how much time has elapsed on my timer

// do more work
console.timeEnd('findDonorsByProgram'); // console log total time and stop timer
```
Downside is this doesn't use the Logger object configured here.

I actually don't recommend leaving timing logs in the code in this form, fine to use when profiling but they are noisy in production logs when really only needed for devs.

Reply:
Probably remove, but you can do it in your next PR, don't need to modify in this RC branch. I'm not looking for edits just reading through.